src/dropdown/bs-dropdown-container.component.ts

```diff
@@ -87,6 +83,14 @@ export class BsDropdownContainerComponent implements OnDestroy {
     return this._element.nativeElement.contains(el);
   }
 
+  ngAfterViewInit(): void {
```

Comment:
@sonicpro I believe this should be after content ivent, because `.dropdown-menu` is in content, not in html of this component